<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BachDisplayOptions</class>
 <widget class="QFrame" name="BachDisplayOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>29</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Thumbnail Size:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="mThumbSizeCombo">
     <item>
      <property name="text">
       <string>256x256</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>512x512</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mImageCheck">
     <property name="text">
      <string>Images</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mMovieCheck">
     <property name="text">
      <string>Movies</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mSequenceCheck">
     <property name="text">
      <string>Sequences</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Display Size (thumbs per line):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="mThumbCountSlider">
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>24</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
     <property name="invertedControls">
      <bool>true</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="mThumbCountSpin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mThumbCountSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mThumbCountSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>761</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>703</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mThumbCountSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mThumbCountSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>6</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
